wiki/Storyboard_Scripting/en.md

```diff
@@ -1,17 +1,20 @@
 # Storyboard scripting
 
-![An example of scripting in .osb.](img/SBS_Base.jpg "An example of scripting in .osb.")
+![An example of scripting in .osb](img/SBS_Base.jpg "An example of scripting in .osb")
 
-In addition to using the [Storyboard Editor](/wiki/Beatmap_Editor/Design)(Design tab) to create [Storyboards](/wiki/Storyboards) for maps, you can also use a plain text editor (such as Notepad or [Notepad++](http://www.notepad-plus-plus.org/)). This is referred to as "**Storyboarding by Scripting**" (SBS), and allows you to be more precise (to the pixel and the millisecond) as to when sprites appear, what they do, and so on.
+In addition to the [built-in storyboard editor](/wiki/Beatmap_Editor/Design) to create [storyboards](/wiki/Storyboards) for maps, a plain text editor, such as Notepad or [Notepad++](http://www.notepad-plus-plus.org/), can also be used. This workflow is called **storyboarding by scripting**, or SBS, and allows one to be more precise (to the pixel and the millisecond) as to when sprites appear, how they behave, and so on.
 
 ## Beginner's guide
 
-[Doomsday has provided a nice beginner's tutorial to the concepts of SBS here (12:22).](http://www.youtube.com/watch?v=UJ1YLDs-bZg)
+An overview of the storyboard scripting concepts by Doomsday: [osu! - Storyboard Scripting for Beginners (12:22)](http://www.youtube.com/watch?v=UJ1YLDs-bZg "YouTube").
 
 ## Full specification
 
-It is primarily based on [the official specifications](https://osu.ppy.sh/community/forums/topics/1869) and experience in making large SBs. This is **not a step-by-step guide**, although simple examples are provided. It is meant to exhaustively detail how SBS translates into actions on-screen in osu!.
+*Note: save your work in the built-in editor first, as it optimizes some things out, and only then in Notepad. **Save often** to prevent accidental progress loss.*
```

Comment:
```suggestion
*Note: Save your work in the built-in editor first, as it optimises some things out, and only then in a plain text editor. **Save often** to prevent accidental progress loss.*
```